ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -369,6 +369,17 @@ objects:
           setup-google-cloud-sdk
         fi
 
+        if [[ "${ENABLE_FIPS}" == true ]]; then
```

Comment:
I think this FIPS check should be in the part of the e2e suite or a helper function folks can call in a job definition.  Having it gated on `ENABLE_FIPS` means you don't have a way to test "when I *don't* set `ENABLE_FIPS`, are my nodes not using FIPS?".  And it also blocks you from running tests where you enable FIPS but for whatever reason (BYO-RHEL?) the usual "is FIPS enabled?" checks don't work on all nodes.  Example of a helper function is `patch_image_specs`, which you can see defined [here][1] and consumed [here][2] (although because our templates are not very DRY, we have duplicate definitions in both `cluster-launch-installer-custom-test-image.yaml` and `cluster-launch-installer-e2e.yaml`).

[1]: https://github.com/openshift/release/blob/4b4f04abe8aa654696929149a157a0dfddbd4565/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L222
[2]: https://github.com/openshift/release/blob/c16b552648d25a1eb2cade0744e27b20b2700e89/ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml#L4326

Reply:
thanks, :eyes:

Reply:
@wking, take another look? I've added a check_fips fn, and added to TEST_COMMAND for e2e-aws, e2e-aws-fips